IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,91 @@
 
+enum Size: Double {
+    case small = 2.40
+    case medium = 3.50
+    case large = 5.00
+}
+
+struct Flavor {
+    var name: String
+    var rating: Float
+    
+    init(name: String, rating: Float) {
+        self.name = name
+        self.rating = rating
+    }
+}
+
+struct Cone {
+    var flavor: String
+    var topping: String?
+    var size: Size
+    
+
+    func eat() {
+        print("chocolate")
+    }
+}
```

Comment:
Here they wanted you to pass in the flavor name so something like "Mmm! I love \(flavor)", that way when you call .eat() on any cone object it will print whatever flavor that cone is, this will always print chocolate.